### PR TITLE
Builder use implemented interfaces properties

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gtk/issues/892

cc @HossainAdnan, @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
Updated with using child configs, also changed many "ignore=true" for properties.

File: src/auto/about_dialog.rs [hunk 1]
```diff
@@ -127,7 +127,6 @@ pub struct AboutDialogBuilder {
     parent: Option<Container>,
     receives_default: Option<bool>,
     sensitive: Option<bool>,
-    //style: /*Unknown type*/,
```
Comment (EPashkin) [inline src/auto/about_dialog.rs]: Deprecated property

**Comment by EPashkin** [comment]
Ready for final review

**Comment by sdroege** [comment]
:+1:

**Comment by EPashkin** [comment]
CI passed

**Comment by EPashkin** [comment]
Ping

File: src/auto/about_dialog.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply src/auto/about_dialog.rs]: Shouldn't we handle it like we do with functions? Like adding the deprecated attribute starting from a given version and remove it from the version it got removed?

**Comment by GuillaumeGomez** [comment]
I just have one question but otherwise looks good to me as well!

File: src/auto/about_dialog.rs (hunk 1 above)
Comment (EPashkin) [reply src/auto/about_dialog.rs]: It was deprecated in 3.0: https://developer.gnome.org/gtk3/stable/GtkStyle.html#gtk-style-new
so it totally deprecated

Comment (GuillaumeGomez) [reply src/auto/about_dialog.rs]: Perfect then!

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**